### PR TITLE
[4.4] translation description text static error pages

### PR DESCRIPTION
**Pull request by heelc29**
replaces #42023 for J4.4-dev

### Summary of Changes

update innerHTML of #descText1 element (translation)

### Testing Instructions

produce an error page are open any static error page (html) directly
eg. `domain.tl/includes/incompatible.html`
switch language

### Actual result BEFORE applying this Pull Request

description text is always english
image

### Expected result AFTER applying this Pull Request

description is translated
image

### Link to documentations
Please select:
- [x] No documentation changes for docs.joomla.org needed
- [x] No documentation changes for manual.joomla.org needed

### CODE REVIEW

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 865a9010cff6bf0cdbbd1c11e0e7e75701fa72bd<br /><br />I've used the following methods to get the different html error pages and then changed the language a few times to different values.

### build_incomplete.html

On a clean branch without having run `composer install` and `npm ci`, go to the site's URL.

### incompatible.html

After having run `composer install` and `npm ci` but before installing Joomla, edit file `installation/index.php` file and change `define('JOOMLA_MINIMUM_PHP', '7.2.5');` to e.g. `define('JOOMLA_MINIMUM_PHP', '9.2.5');`.

Then go to the site's URL.

### fatal-error.html

After successful installation, delete all Joomla tables in the database of this site e.g. with phpMyAdmin and then go to the site's URL.
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/42030">issues.joomla.org/tracker/joomla-cms/42030</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 865a9010cff6bf0cdbbd1c11e0e7e75701fa72bd<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/42030">issues.joomla.org/tracker/joomla-cms/42030</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/42030">issues.joomla.org/tracker/joomla-cms/42030</a>.</sub>

File: build/warning_page/template.js [hunk 1]
```diff
@@ -18,20 +20,19 @@ var errorLocale = window.errorLocale || null;
       }
 
       document.getElementById('translatedLanguagesSelect').addEventListener('change', function(e) {
-        var ref = e.target.value, helpLink, meta;
+        var ref = e.target.value;
 
         if (ref) {
```
Comment (laoneo) [inline build/warning_page/template.js]: Can you invert here the condition and return early?

**Review by laoneo — CHANGES_REQUESTED** [review]

File: build/warning_page/template.js (hunk 1 above)
Comment (heelc29) [reply build/warning_page/template.js]: Done. There is no modification in the generated files due minify command in build tools.
https://github.com/joomla/joomla-cms/blob/a390ec1e8aaafc22a67aca4ce5d418646d7c9e70/build/build-modules-js/error-pages.es6.js#L41

File: build/warning_page/template.js [hunk 2]
```diff
@@ -46,15 +49,13 @@ var errorLocale = window.errorLocale || null;
...
-
-        if (helpLink) {
+        if (header && desc1 && helpLink) {
+          header.innerHTML = errorLocale[key].header;
+          desc1.innerHTML = errorLocale[key].text1;
           helpLink.innerText = errorLocale[key]['help-url-text'];
         }
 
-        meta = document.querySelector('[http-equiv="Content-Language"]');
+        var meta = document.querySelector('[http-equiv="Content-Language"]');
```
Comment (laoneo) [inline build/warning_page/template.js:58]: Do we need here to do "var meta" again as it is already done above?

**Comment by laoneo** [comment]
Can you fix the conflicts and respond to my question? Would love to see this one in 4.4.1.

File: build/warning_page/template.js (hunk 2 above)
Comment (heelc29) [reply build/warning_page/template.js:58]: Yes I think so. I moved it only from line 51:
image

If I remove it it produce an js error:
image

**Comment by heelc29** [comment]
> Can you fix the conflicts and respond to my question?

Done

File: build/warning_page/template.js (hunk 2 above)
Comment (laoneo) [reply build/warning_page/template.js:58]: Thanks!

**Review by laoneo — APPROVED** [review]

**Comment by laoneo** [comment]
Thanks!

**Pull request merged**